### ISSUE TITLE
jellyfin couldn't fetch scenes

### ISSUE THREAD

**Issue by maaya1989** [ThePornDatabase/Jellyfin.Plugin.ThePornDB#63]
Recently switched to jellyfin, the plugin worked fine on emby and plex, all scenes were fetched perfectly somehow it didn't work on jellyfin, it couldn't fetch any content in the folder. Strangely the movie part worked fine too. Btw I'm using the development build.

**Comment by DirtyRacer1337** [comment]
Strange code base absolutely the same to emby version.

You can try to disable `UseFilePath` or `UseOSHash` in plugin setting, that's the only difference

**Comment by KrisNie** [comment]
I have the same issue.

I think there is something different about metadataapi.net. They have `scenes/`  & `movies/` & `JAV`... ...
But this plugin seems like only tries to fetch `movies/`.
In the Identify function it cannot find anything of metadataapi.net.

Hope that helps, really like this type of plugin!

**Comment by Schievezot** [comment]
> I have the same issue.
> 
> I think there is something different about metadataapi.net. They have `scenes/` & `movies/` & `JAV`... ... But this plugin seems like only tries to fetch `movies/`. In the Identify function it cannot find anything of metadataapi.net.
> 
> Hope that helps, really like this type of plugin!

On my server by default the Identify function apparently searches in /scenes while it should search in /movies...

**Comment by DirtyRacer1337** [comment]
Use dev build and just select right provider, there is 3 for every type

**Comment by Schievezot** [comment]
> Use dev build and just select right provider, there is 3 for every type

Yeah, but I have no idea how to install it on my synology... Can't use Docker... Could it be possible to add the Dev build to the repository?

**Comment by hutese** [comment]
I'm having the exact same issue. Jellyfin can fetch movies no problem, but scenes just come up blank. Even tried manually putting in the ID into "identify" but nothing comes up. Have used both latest build and dev build. What am I doing wrong?

**Comment by KrisNie** [comment]
> Use dev build and just select right provider, there is 3 for every type

After inputting the token, I still can not identify the metadata.
